### PR TITLE
Simplify BMW flow

### PR DESCRIPTION
**Pull request by JMMackenzie**
Clean up for #281 

Changes proposed in this pull request:

- Minor simplification of BMW flow control

### CODE REVIEW

**Review by elshize — APPROVED** [review]
Looks good to me. @amallia?

**Comment by amallia** [comment]
I will review it in the next 24h. Currently traveling to NYC

**Comment by elshize** [comment]
Sure, take your time.

File: include/pisa/query/algorithm/block_max_wand_query.hpp [hunk 1]
```diff
@@ -134,26 +134,20 @@ struct block_max_wand_query {
                     }
                 }
 
-                uint64_t next_jump = max_docid;
-
-                if (pivot + 1 < ordered_cursors.size()) {
```
Comment (amallia) [inline include/pisa/query/algorithm/block_max_wand_query.hpp]: Why do you want to remove this block? I think we will be missing something if you do so. I can’t explain more as I am from my phone

**Review by amallia — APPROVED** [review]

**Comment by amallia** [comment]
Actually it was easy to review from my phone

**Pull request merged**